packages/web-components/src/presence-badge/svg/svg.ts

```diff
@@ -0,0 +1,38 @@
+// © Microsoft Corporation. All rights reserved.
+
+// FILLED ICONS:
+
+/** Status: Available indicated by a check mark in a filled circle */
+export const statusAvailableGlyph = `<svg fill="none" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m8 16c4.4183 0 8-3.5817 8-8 0-4.41828-3.5817-8-8-8-4.41828 0-8 3.58172-8 8 0 4.4183 3.58172 8 8 8zm3.7069-9.29261-4.00329 4.00001c-.18766.1875-.44214.2928-.70742.2926-.26529-.0002-.51964-.1057-.70707-.2935l-1.99679-1.99996c-.39021-.39084-.38971-1.024.00113-1.41421s1.024-.38971 1.41421.00113l1.28998 1.29205 3.29555-3.2929c.3907-.39037 1.0239-.39011 1.4142.00057.3904.39068.3901 1.02385-.0005 1.41421z"></svg>`;
```

Comment:
Did we trim the padding from these? Originally 20px icons? Is there a way to keep the padding and support the design. Many icon libraries, including Fluent, have built in padding, so if we expect none it makes them hard to swap out.

Also we should optimize these in [svgomg](https://jakearchibald.github.io/svgomg/), precision 2 is what I use.

Reply:
Actually, we should import from [fluent system icons](https://github.com/microsoft/fluentui-system-icons/tree/master/packages/svg-icons) if possible.